2022q4/Makefile

```diff
@@ -8,34 +8,38 @@ YEAR?=	2022
 QUARTER?=	4
 
 # Entries from the various official and semi-official teams
-TeamReports= freebsd-foundation.adoc releng.adoc clusteradm.adoc ci.adoc portmgr.adoc
+TeamReports= freebsd-foundation.adoc releng.adoc clusteradm.adoc ci.adoc portmgr.adoc status.adoc
 
 # Projects that span multiple categories, from the kernel, and userspace to
 # the Ports Collection or external projects
-Projects=
+Projects= vessel-status-report.adoc NFS-in-prisons.adoc
 
 # Changes affecting the base system and programs in it
-Userland=
+Userland= openssh.adoc
 
 # Updates to kernel subsystems/features, driver support, filesystems,
 # and more
-Kernel=
+Kernel= ufs_snapshots.adoc ddb_ctf.adoc
 
 # Updating platform-specific features and bringing in support for new
 # hardware platforms
-Architectures=
+Architectures= golang_riscv64.adoc
+
+# Updating cloud-specific features and bringing in support for new
+# cloud platforms
```

Comment:
This comment will become the description of the new Cloud category, so please make sure to review it. As you might notice, I have taken the Architectures category description and edited it to apply to the cloud. Feel free to suggest a better description.

Reply:
@pauamma I have done something similar to what you suggested: copied comment text and resolved the conversation on the wrong lines. Thanks for your help.